test/load/allocation/README.md

```diff
@@ -0,0 +1,10 @@
+This test creates a proper amount of GameServers at the initial step, switch them to Allocated state and finally shut them down (`automaticShutdownDelayMin` flag in a simple-udp).
+
+1) Run kubectl apply -f ./fleet.yaml
+2) Run `runAllocation.sh` script to perform this test. You can provide a number of runs as a parameter (3 is a default value)
+
+Number of allocated GameServers = numberOfClients * reqPerClient (see `allocationload.go`)
```

Comment:
Sorry, I'm not sure what this means.

Reply:
This is used in the line below, intended to clarify parameters to pass to the test.